nodeJS/apis/api_security.md

```diff
@@ -22,14 +22,16 @@ This strategy, while particularly useful with APIs can be used with a traditiona
 <div class="lesson-content__panel" markdown="1">
 
 1. [This video](https://www.youtube.com/watch?v=7nafaH9SddU) is a great resource that explains everything you need to know about creating and verifying JSON Web Tokens.
-2. [This article](https://dev.to/_arpy/learn-using-jwt-with-passport-authentication-22n8) covers setting up JSON Web Tokens within the PassportJS system that you should already be familiar with. In this case you don't _really_ need Passport, but it does handle quite a bit of stuff that the guy in the video set up manually.
+2. [This article](https://dev.to/_arpy/learn-using-jwt-with-passport-authentication-22n8) covers setting up JSON Web Tokens within the PassportJS system that you should already be familiar with. In this case you don't _really_ need Passport, but it does handle quite a bit of stuff that the guy in the video set up manually. NOTE: There is some typos in the code blocks provided in the article, its good exercise to code along and spot them ;). Aswell, some function are now deprecated and this needs some extra reading of documentation in order to make it work.
```

Comment:
I think with this one.. it would be a better idea to just find a better article than to add all of these disclaimers.

Reply:
I think so yes :)
So I've updated with a video from Web Dev Simplified, for its clarity.